### PR TITLE
added mixpanel to track who opened window

### PR DESCRIPTION
**Pull request by amishyn**
- added mixpanel to record room and username who opened it

note: how to run tests ? there is no test section in readme.

### CODE REVIEW

File: web/static/js/views/room.js [hunk 1]
```diff
@@ -24,6 +25,11 @@ class Room extends Marionette.LayoutView {
   }
 
   onRender() {
+    mixpanel.track("Room opened", {
```
Comment (grafmolnia) [inline web/static/js/views/room.js]: I'd prefer to put tracking code after main action (after `this.leftRegion.show` and `this.rightRegion.show` here).

Comment (grafmolnia) [inline web/static/js/views/room.js]: There is `mixpanel is not defined` error in dev mode.

**Comment by le0pard** [comment]
No need pass to views not needed value and use behavior for tracking http://marionettejs.com/docs/marionette.behavior.html

**Comment by le0pard** [comment]
You need to define http://marionettejs.com/docs/v2.4.4/marionette.behaviors.html#behaviorslookup

File: web/static/js/views/room.js [hunk 2]
```diff
@@ -41,6 +43,14 @@ class Room extends Marionette.LayoutView {
       rightRegion: '#rightRegion'
     }
   }
+
+  behaviors: {
+    TrackShow: {
+      name: 'Room shown',
+      roomId:  this.roomId,
```
Comment (le0pard) [inline web/static/js/views/room.js]: I think this will be undefined here. Did you check?

File: web/static/js/behaviours/index.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import TrackShow from '../track_show';
+
```
Comment (le0pard) [inline web/static/js/behaviours/index.js]:
```
import TrackShow from '../track_show';

const Behaviors = {
  TrackShow
};

Marionette.Behaviors.behaviorsLookup = function() {
  return Behaviors;
};
```

@amishyn what do you think?

Comment (amishyn) [inline web/static/js/behaviours/index.js]: lets do it

**Pull request merged**